GA/config.py

```
@@ -0,0 +1,302 @@
+from Submodels.Parameterizations import AirfoilParameterization # type: ignore
+
+# Define the seed used for randomisation
+GLOBAL_SEED = 42
+
+# Define the objective IDs used to construct the objective functions
+class ObjectiveID(IntEnum):
+    """
+    Enumeration of objective function identifiers for the optimization problem.
+    Each member represents a different optimization objective that can be used in the genetic algorithm's fitness evaluation.
+    """
+    def _generate_next_value_(name, start, count, last_values):
```

Pylint (3.3.7)

[error] Method '_generate_next_value_' should have "self" as first argument

(E0213)